src/Common/Sanitizer/SanitizerTelemetry.cs

```diff
@@ -29,11 +31,17 @@ public class SanitizerTelemetry
 
         public TimeSpan SanitizeDuration { get; set; }
 
+        public SanitizerTelemetry(bool showSecretsWarning)
```

Comment:
This is technically a breaking change, because the default constructor (the one without parameters) will be hidden by this. Are we sure about the backward compatibility?

Reply:
The change was due to the value in the telemetry may be not consistent with the DisplaySecretsWarning option. Previously the ShowSecretsWarning in the telemetry relied on if the real detection method was invoked. If not, then the value would be the default (false). However, the real value of DisplaySecretsWarning may be true. The initialization of the SanitizerTelemetry just happens in Az.Accounts module (For SDK, Authentication proj; for AutoRest, AzModule). So it won't cause any real breaking change and build error.